packages/react/src/components/Storage/FileUploader/Previewer/__tests__/Previewer.test.tsx

```diff
@@ -27,82 +27,144 @@ const fileStatus2: FileStatus = {
 };
 const fileStatuses: FileStatuses = [fileStatus];
 const fileStatuses2: FileStatuses = [fileStatus, fileStatus2];
+const ref = {
+  current: {
+    info: jest.fn(),
+  },
+} as unknown as React.MutableRefObject<HTMLInputElement>;
+
+const commonProps = {
+  acceptedFileTypes: ['.png'],
+  fileStatuses: fileStatuses,
+  inDropZone: false,
+  isEditingName: [],
+  isLoading: false,
+  isSuccess: false,
+  maxFilesError: false,
+  multiple: true,
+  onClear: () => null,
+  onDragEnter: () => null,
+  onDragLeave: () => null,
+  onDragOver: () => null,
+  onDragStart: () => null,
+  onDrop: () => null,
+  onFileChange: () => null,
+  onFileClick: () => null,
+  percentage: 0,
+  hiddenInput: ref,
```

Comment:
Rather than copying these over and over again everywhere, I created an object so I can spread them wherever I need and just show the changes inside the test.